src/api/json/catalog.json

```diff
@@ -1109,13 +1109,18 @@
       "url": "https://raw.githubusercontent.com/iterative/dvcyaml-schema/master/schema.json"
     },
     {
-      "name": "Eclipse Che Devfile",
-      "description": "JSON schema for Eclipse Che Devfiles",
-      "url": "https://raw.githubusercontent.com/eclipse-che/che-server/master/wsmaster/che-core-api-workspace/src/main/resources/schema/1.0.0/devfile.json",
+      "name": "Devfile",
+      "description": "JSON schema for Devfiles",
+      "url": "https://raw.githubusercontent.com/devfile/api/2.1.x/schemas/latest/devfile.json",
       "fileMatch": [
         "devfile.yaml",
         ".devfile.yaml"
-      ]
+      ],
+      "versions": {
+        "1.0.0": "https://raw.githubusercontent.com/eclipse-che/che-server/master/wsmaster/che-core-api-workspace/src/main/resources/schema/1.0.0/devfile.json",
```

Comment:
is it expected that master branch is used?

Reply:
I've left the 1.0.0 link as it is, since this is maintained by a different github org(& team) and hosted on repo `eclipse-che/che-server` and they have a different release process as compared to devfile version 2.0.0 + which are hosted on repo `devfile/api`